docker-app/qfieldcloud/core/utils2/sentry.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from io import StringIO
+
+from sentry_sdk import configure_scope
+
+logger = logging.getLogger(__name__)
+
+
+def report_to_sentry(
+    name: str, pre_serialization: str, post_serialization: str, buffer: StringIO
+):
+    """
+    QF-2540
+    Send the report to Sentry
```

Comment:
Upfront: sorry for nitpicking on naming ;-)

Since this is very specific for QF-2540, I think we should also reflect that in the name (`report_serialization_diff_to_sentry` or similar), the current name suggests that it's quite generic.

Sidenote: I appreciate doc strings that add more than what's visible from the function name. If they don't add more then it's better to just leave them. But even better than that is adding docstrings with param and return value documentation, a bit more explanation (e.g. will add "x and y as an attachment" in this case here)

Reply:
Please also have a look at the sidenote

Reply:
Hovering over `report_serialization_diff_to_sentry` shows:

![image](https://github.com/opengisch/qfieldcloud/assets/19554213/a3c78848-1dd2-4beb-b980-31aec0c28514)

not good enough?